laurent22/joplin
# Desktop: Fixes #9737: Fix ghosting when dropping code block over another code block

Fixes https://github.com/laurent22/joplin/issues/9737

When dragging a code block if it was released over another code block, a "ghost" of the element would appear over where the mouse was released and it wouldn't go away.

The problem was that TinyMCE was throwing an error because of the type of attribute used by the `code` element.

**Important:** This change means that code blocks aren't "draggable" in the Rich text editor anymore. I don't think this was ever the intention and was a behavior caused by the bug. Since it appears this was to be the only type of content in the rich text editor that was draggable I don't think it will matter too much (content can still be moved around by copying/cutting)


### Testing

- Create a code block in the rich text editor
- Drag the code block and release over the original location
- Previously, the ghost of the element that was being dragged would appear over the code block, now, nothing should change
